meta-mbl-distro/recipes-core/base-files/base-files_%.bbappend

```diff
@@ -57,4 +78,14 @@ do_install_append() {
 ${FSTAB_LINES}
 EOF
     cat ${D}${sysconfdir}/fstab
+
+    # Install the partition data files in the factory config partition
+    # This information is needed for component updates.
+
+    install -d "${D}${MBL_FACTORY_CONFIG_MOUNT_POINT}/part-infos"
+
+    for file in "${MBL_PARTITION_DATA_FILES}"
+    do
+        install -m 0644 $file "${D}${MBL_FACTORY_CONFIG_MOUNT_POINT}/part-infos"
```

Comment:
I suppose the partition offset information file is ro, correct? If so it should be installed with `0444`.